### PR TITLE
feature: remove GVol-BTC-IV and GVol-ETH-IV

### PR DESCRIPTION
**Pull request by matheushahnn**

### CODE REVIEW

**Comment by ramos-ph** [comment]
Are we still using those backup files? I thought we had deleted them.

File: backup/prod/feeds.yaml [hunk 1]
```diff
@@ -5314,128 +5314,6 @@ AMPL-USD-VWAP-1day:
       calculator:
         name: VWAP
 
-# GVol
```
Comment (ramos-ph) [inline backup/prod/feeds.yaml:5317]: This file shouldn't be here. It's a backup file under /prod.

Comment (matheushahnn) [reply backup/prod/feeds.yaml:5317]: I'll remove this file.

Comment (matheushahnn) [reply backup/prod/feeds.yaml:5317]: This file is created when we run the command `npm run script:backupFeeds` not sure if it is right to remove the file.

**Review by ramos-ph — APPROVED** [review]

**Pull request merged**